src/main/scala/com/typesafe/sbt/packager/docker/Keys.scala

```diff
@@ -38,6 +38,7 @@ trait DockerKeys {
   )
   val dockerExecCommand = SettingKey[Seq[String]]("dockerExecCommand", "The shell command used to exec Docker")
   val dockerVersion = TaskKey[Option[DockerVersion]]("dockerVersion", "The docker server version")
+  val dockerApiVersion = TaskKey[Option[DockerApiVersion]]("dockerApiVersion", "The docker server api version")
```

Comment:
Can you move this into the trait `DockerKeysEx` below.  This will resolve the binary incompatibility issues :smiley:

Reply:
thanks!